test/drawers/drawerTests.ts

```diff
@@ -20,10 +20,12 @@ class MockAnimator implements Plottable.Animators.Plot {
   }
 }
 
-class MockDrawer extends Plottable.Drawer {
-  public _drawStep(step: Plottable.Drawers.AppliedDrawStep) {
-    step.animator.animate(this.renderArea(), step.attrToAppliedProjector);
-  }
+function getMockDrawer(dataset: Plottable.Dataset) {
```

Comment:
Note that this mock drawer specifically mocks out the `drawStep` function.  Does that mean that if other tests wanted to use a mock drawer, should they always retrieve this one that has its `drawStep` mocked out? or should this function be renamed to more hint at the fact that the `drawStep` function is mocked out?